core/papersize.lua

```diff
@@ -58,13 +58,14 @@ setmetatable(papersize, {
     __call = function (self, size)
       local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
```

Comment:
Why %s+ and not %s* ?

Reply:
Because some paper size presets have an `x` in them such as "executive" and "halfexecutive". Unless we switch to a PEG based parser that detects valid iterations of our units format we need something more definitive than just an X to split this field.

Reply:
It can wait for sure. I only asked as I've tried `5cmx5cm` before.